### PR TITLE
Add Subscription pricing table

### PR DESCRIPTION
**Pull request by Zishan-7**
TODO:
- ~~Catchup with Clem for monthly plan details~~

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ import NotFound from './shared/NotFound';
 import OperatorDetails from './operators/OperatorDetails';
 import OperatorList from './operators/OperatorList';
 import { ServiceProvider } from './@services/ServiceContext';
+import SubscriptionPlans from './subscription/SubscriptionPlans';
```
Comment (rubo) [inline src/App.jsx:21]:
```suggestion
import PrivateOutlet from './shared/PrivateOutlet';
import { ServiceProvider } from './@services/ServiceContext';
import SubscriptionPlans from './subscription/SubscriptionPlans';
```

File: src/App.jsx [hunk 2]
```diff
@@ -30,6 +31,7 @@ const router = createBrowserRouter(
       <Route element={<OperatorList />} path="/operators" />
       <Route element={<OperatorDetails />} path="/operators/:address/:tab?" />
       <Route element={<LST />} path="/lst" />
+      <Route element={<SubscriptionPlans />} path="/subscriptions" />
```
Comment (rubo) [inline src/App.jsx:34]: Making the route available for logged in users only:

```suggestion
      <Route element={<PrivateOutlet />} path="/subscriptions">
        <Route element={<SubscriptionPlans />} index />
      </Route>
```

**Review by vincenthongzy — APPROVED** [review]
Is there design for how to handle the success callback? Like showing an actual success page that user managed to subscribe?

**Comment by Zishan-7** [comment]
> Is there design for how to handle the success callback? Like showing an actual success page that user managed to subscribe?

There will be a thank you page. Still needs to be designed

**Pull request merged**